z3/snap.py

```diff
@@ -211,6 +213,12 @@ def list(self):
         return self._snapshots.values()
 
     def get_latest(self):
+        if len(self._snapshots) == 0:
+            cfg = get_config()
```

Comment:
Basically I try to avoid import-time side effects, since it's made debugging harder for me in the past.